admin/load.php

```diff
@@ -336,3 +336,28 @@ function perflab_get_module_data( $module_file ) {
 
 	return $module_data;
 }
+
+/**
+ * Enable all non-experimental modules on plugin activation.
+ *
+ * @since 1.0.0
+ */
+function perflab_activation_hook() {
+	// Bail if option is already set with any value.
+	if ( false !== get_option( PERFLAB_MODULES_SETTING, false ) ) {
+		return;
+	}
```

Comment:
```suggestion
	if ( ! get_option( PERFLAB_MODULES_SETTING, false ) ) {
		return;
	}
```

Looks like the value being stored is never false but it can be an empty array instead checking for `falsy` value should be enough

Reply:
@mitogh The code suggestion here seems incorrect since now the code won't set the option if it's not set. Did you mean the reverse here?

The intention here is to check whether no value is set for the option. If the option is set to an empty array, we shouldn't do anything here. Only if the option isn't set at all, the logic here should be run. So in this case, getting an empty array from the database is different than getting an empty array from the option's default value.

Reply:
Good catch. I was intended to suggest the opposite, _let me update my suggestion._